modules/editorial-metadata/editorial-metadata.php

```diff
@@ -327,1114 +328,1203 @@ function register_taxonomy() {
 						'add_new_item' => __( 'Add New Editorial Metadata', 'edit-flow' ),
 						'new_item_name' => __( 'New Editorial Metadata', 'edit-flow' ),
 					),
-				'rewrite' => false,
-			)
-		);
-	}
-	
-	/*****************************************************
-	 * Post meta box generation and processing
-	 ****************************************************/
-	
-	/**
-	 * Load the post metaboxes for all of the post types that are supported
-	 */
-	function handle_post_metaboxes() {
-		$title = __( 'Editorial Metadata', 'edit-flow' );
+					'rewrite' => false,
+				)
+			);
+		}
 
-		$supported_post_types = $this->get_post_types_for_module( $this->module );
-		foreach ( $supported_post_types as $post_type ) {
-			add_meta_box( self::metadata_taxonomy, $title, array( $this, 'display_meta_box' ), $post_type, 'side' );
+		/*****************************************************
+		 * Post meta box generation and processing
+		 ****************************************************/
+
+		/**
+		 * Load the post metaboxes for all of the post types that are supported
+		 */
+		public function handle_post_metaboxes() {
+			$title = __( 'Editorial Metadata', 'edit-flow' );
+
+			$supported_post_types = $this->get_post_types_for_module( $this->module );
+			foreach ( $supported_post_types as $post_type ) {
+				add_meta_box( self::metadata_taxonomy, $title, array( $this, 'display_meta_box' ), $post_type, 'side' );
+			}
 		}
-	}
-	
-	/**
-	 * Displays HTML output for Editorial Metadata post meta box
-	 *
-	 * @param object $post Current post
-	 */
-	function display_meta_box( $post ) {
-		echo "<div id='" . self::metadata_taxonomy . "_meta_box'>";
-		// Add nonce for verification upon save
-		echo "<input type='hidden' name='" . self::metadata_taxonomy . "_nonce' value='" . wp_create_nonce( 'ef-save-metabox' ) . "' />";
-
-		if ( current_user_can( 'manage_options' ) ) {
-			// Make the metabox title include a link to edit the Editorial Metadata terms. Logic similar to how Core dashboard widgets work.
-			$url = add_query_arg( 'page', 'ef-editorial-metadata-settings', get_admin_url( null, 'admin.php' ) );
-			echo '<p><a href="'. esc_url( $url ) . '">' . __( 'Configure', 'edit-flow' ) . '</a></p>';
+
+		/**
+		 * Displays HTML output for Editorial Metadata post meta box
+		 *
+		 * @param object $post Current post
+		 */
+		public function display_meta_box( $post ) {
+			echo "<div id='" . esc_attr( self::metadata_taxonomy ) . "_meta_box'>";
+			// Add nonce for verification upon save
+			echo "<input type='hidden' name='" . esc_attr( self::metadata_taxonomy ) . "_nonce' value='" . esc_js( wp_create_nonce( 'ef-save-metabox' ) ) . "' />";
```

Comment:
I think `esc_attr()` is more appropriate for a nonce value, which is just a alpha-numeric string.

```suggestion
			echo "<input type='hidden' name='" . esc_attr( self::metadata_taxonomy ) . "_nonce' value='" . esc_attr( wp_create_nonce( 'ef-save-metabox' ) ) . "' />";
```

Reply:
Done